src/components/zen-avatar-icon/zen-avatar-icon.tsx

```diff
@@ -48,8 +48,10 @@ export class ZenAvatarIcon {
         // Get initials oly from name
         initials = this.userName.substring(0, 2).toUpperCase();
       }
-    } else {
+    } else if (this.email && this.email.trim() != '') {
       initials = this.email.substring(0, 2).toUpperCase();
```

Comment:
```suggestion
    } else if (this.email.trim()) {
      initials = this.email.trim().substring(0, 2).toUpperCase();
```

Yep, it's repeating the `.trim()`, but if you validate the string with `trim` and then do the `substring` without using `trim`, you could end up with a space.

Reply:
Good catch. But not because of the shorter code.  But because considering the imageUrl as well. Done